### DISCUSSION TITLE
I'm a software developer

### DISCUSSION THREAD

**Discussion by v8hid** [WeAreMahsaAmini/Core#35] [Introductions]
My name is Vahid, full stack developer, I can help on multiple fields including : dev-ops, backend , frontend(web) 
I have many skills but I prefer helping on server side stuff, I'm available btw.

**Comment by siamakhaz** [comment]
Hi, my name is Siamak I can help with DevOps and server-side stuff.

**Reply by anonymoper** [reply]
Nice to have you on the team! 

Please let me know what you think about this idea [https://github.com/WeAreMahsaAmini/Core/discussions/66](https://github.com/WeAreMahsaAmini/Core/discussions/66)
Thank you!
